third_party/xla/third_party/compute_library/acl_gemm_scheduling_heuristic.patch

```diff
@@ -0,0 +1,147 @@
+# *******************************************************************************
+# Copyright 2025 Arm Limited and affiliates.
+# SPDX-License-Identifier: Apache-2.0
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# *******************************************************************************
```

Comment:
Please use the OpenXLA copyright header instead.
https://github.com/openxla/xla/blob/c448ed81b004a48302a798806e7449d4ef22765f/xla/python/xla_extension.py#L1-L14

Reply:
Would I be able to add an AUTHORS file to `third_party/xla` in the same vein as `tensorflow/AUTHORS` as part of this patch?